systers_portal/users/forms.py

```diff
@@ -7,6 +7,13 @@
 
 class UserForm(forms.ModelForm):
     """User form combined with SystersUserForm"""
+    first_name = forms.CharField(max_length=20,
+                                 help_text='Maximum characters allowed: 20',
+                                 required=False)
+    last_name = forms.CharField(max_length=20,
+                                help_text='Maximum characters allowed: 20',
+                                required=False)
+
```

Comment:
👋 Did you run tests after this?

Reply:
Yes, I did.
Initially, it was showing some errors but after adding `required=False`, it works well for me.

Reply:
Can you please check with fresh `makemigrations` and `migrate`?